src/features/app/App.js

```diff
@@ -4,6 +4,7 @@ import {
   Row
 } from 'react-bootstrap'
 import { Provider } from 'react-redux'
+import './ReactGA.js'
```

Comment:
Good tip. I only made that because I need it to initialize first, and if I put it in this file, it would complain that some of the components were registering events before the initialize.